docs/FAQ.adoc

```diff
@@ -9,3 +9,8 @@ UPD workarounded in latest versions.
 
 Please see https://github.com/KostyaSha/github-integration-plugin/issues/60#issuecomment-200060734
 Example of workaround: https://gist.github.com/kad/4a5a8f669d4a4090b43be1f1c5461df3
+
+==== Webhook is recieved, but the build is not triggered
```

Comment:
`Something not triggered` not only webhook related, cron run may not see fresh data because of wrong cache

Reply:
Updated, is it better now?

Reply:
Branch also will be affected. All requests  (almost) to GH are cached in connector, so more exact is: `Jenkins doesn't see changes in githhub (i.e. builds not triggered when hook received)`.
I will check how to add fresh okhttp, maybe it will help in future

Reply:
Thank you, updated.